runtime/polkadot/src/lib.rs

```diff
@@ -465,6 +483,7 @@ impl pallet_staking::Config for Runtime {
 	type NextNewSession = Session;
 	type ElectionProvider = ElectionProviderMultiPhase;
 	type GenesisElectionProvider = runtime_common::elections::GenesisElectionOf<Self>;
+	type SortedListProvider = pallet_staking::UseNominatorsMap<Runtime>;
```

Comment:
except this line, no other change to polkadot runtime please.

Reply:
we also still need VoterSnapshotperBlock